README.md

```diff
@@ -47,21 +47,21 @@ Use meson to configure the build you want to have. A list of the
 what you want. If you use directories starting with 'build' they will be ignored by git.
 Examples:
 
-        meson setup build
-        meson setup -D tests=false build_without_tests
-        meson setup --prefix=/usr --buildtype=release build_rel
-        meson setup --buildtype=release -D unity=on build_rel_unity
-        meson setup --buildtype=release -D docs=false build_nodocs_rel
-        meson setup -Dcpp_args='-Wconversion -Wshadow' --buildtype=debug builddir
-        CXX=clang++ meson setup build_clang
+        meson setup <build-dir>
+        meson setup -D tests=false <build-without-tests-dir>
+        meson setup --prefix=/usr -D buildtype=release <build-rel-dir>
+        meson setup -D buildtype=release -D unity=on <build-rel-unity-dir>
+        meson setup -D buildtype=release -D docs=false <build-nodocs-rel-dir>
+        meson setup -Dcpp_args='-Wconversion -Wshadow' --buildtype=debug <build-with-extra-warnings-dir>
```

Comment:
Hmmm... is `-D buildtype` the same as `--buildtype`? And why is is not changed in this line?

Reply:
<img width="897" alt="Screenshot 2024-02-17 at 01 11 28" src="https://github.com/gul-cpp/gul14/assets/16012374/9253a30a-2cdb-42b6-8c78-56fa80f81bd9">

So here we have mentioned `-Dbuildtype=release` and `--buildtype=release`. In fact `-D buildtype=release` is **not** mentioned here.

When you type `meson setup --help` all the direct (without `-D`) options are listed.
For that reason
* I would prefer them here, where possible (`buildtype`, `unity`)
* use the `-D` only where needed
* not use `-D opt=value` _at all_ (yes it is documented, but not here)

Reply:
Valid points, I added the `-D` options because I experimented with [`muon`](https://muon.build/) and there all options are used with `-D`. But we don't mention `muon`, so we should tick with meson style.  

I will change it back.